Objects/cown.c

```diff
@@ -205,10 +219,14 @@ static PyObject *PyCown_set_unchecked(PyCownObject *self, PyObject *arg) {
 
         // Tell the region that it is owned by a cown,
         // to enable it to release the cown on close
-        if (is_region_object) {
+        if (arg_is_region_object) {
             _PyRegion_set_cown_parent(arg, _PyObject_CAST(self));
+            // TODO: should not run try close here unless dirty at the end of phase 3
```

Comment:
```suggestion
            // TODO: Pyrona: should not run try close here unless dirty at the end of phase 3
```

Reply:
Ah, I had not realised (or maybe I'd forgotten) that this was the format we were using!